Makefile

```diff
@@ -0,0 +1,22 @@
+
+all: klaw api
+
+klaw:
+	mvn clean install
+	mvn clean package
+
+api:
+	cd cluster-api && mvn clean install
+	cd cluster-api && mvn clean package
+
+edit-config:
+	${EDITOR} target/classes/application.properties
+
+edit-api-config:
+	${EDITOR} cluster-api/target/classes/application.properties
+
+run:
+	java -jar target/klaw-1.0.0.jar
+
+run-api:
+	java -jar cluster-api/target/klaw-clusterapi-1.0.0.jar --spring.config.location=cluster-api/target/classes/application.properties
```

Comment:
I think this is pretty useful, but I doubt it is "idiomatic" in the Java world. I don't think many people using Java projects would run `make`. That being said, I find this more portable than the common `sh` and `bat` files.

Reply:
It can be converted to scripts within a gradle file if it is preferred. This is just cleaner and, as you said, more portable

Reply:
Yes, I really like `make` and I think we should keep it